src/svgcontext.ts

```diff
@@ -118,6 +123,8 @@ export class SVGContext extends RenderContext {
   parent: SVGGElement;
   // The stack of groups.
   groups: SVGGElement[];
+  // The stack of attributes associated with each group.
+  protected groupAttributes: Attributes[];
```

Comment:
`groupAttributes` is a stack that contains the current attributes in each nested group. The active ones, this means, the ones in the group and the ones inhereted from the other groups in the stack.